reload.go

```
@@ -151,6 +154,35 @@ func Do(log func(string, ...interface{}), additional ...dir) error {
 	return nil
 }
 
+func expandAdditional(log func(string, ...interface{}), additional []dir) []dir {
+	result := make([]dir, 0, len(additional))
+	for _, a := range additional {
+		if strings.HasSuffix(a.path, "\\...") {
+			result = append(result, listDirs(log, strings.TrimSuffix(a.path, "\\..."), a.cb)...)
+		} else if strings.HasSuffix(a.path, "/...") {
+			result = append(result, listDirs(log, strings.TrimSuffix(a.path, "/..."), a.cb)...)
+		} else {
+			result = append(result, a)
+		}
+	}
+	return result
+}
+
+func listDirs(log func(string, ...interface{}), rootPath string, cb func()) (result []dir) {
+	filepath.WalkDir(rootPath, func(dirPath string, d fs.DirEntry, err error) error {
+		if err != nil {
+			log("Error reading %s, skipped", dirPath)
```

IMHO it's better to return an error on this (from reload.Do()), rather than skipping it. It's on startup, and skipping directories (even with a print) is somewhat confusing, IMHO.